File: app/src/main/java/com/github/hurshi/MainActivity.java
package com.github.hurshi;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import hurshi.github.com.daggervminjector_annotations.DaggerVMModule;


@DaggerVMModule(TempModule.class)
public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: app_kotlin/src/main/java/com/github/hurshi/app_kotlin/MainActivity.kt
package com.github.hurshi.app_kotlin

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        //        setContentView(R.layout.activity_main);

        val mainViewModel = MainViewModel(application)
        mainViewModel.dogString()
    }
}
